pkg/volume/git_repo/git_repo.go

```diff
@@ -126,10 +126,8 @@ func (b *gitRepoVolumeBuilder) SetUp() error {
 	return b.SetUpAt(b.GetPath())
 }
 
-// This is the spec for the volume that this plugin wraps.
-var wrappedVolumeSpec = &volume.Spec{
-	Volume: &api.Volume{VolumeSource: api.VolumeSource{EmptyDir: &api.EmptyDirVolumeSource{}}},
-}
+// This is the name for the volume that this plugin wraps, will be generated with random value in SetUpAt(..).
+var wrapperVolumeName string
```

Comment:
We could even embed that into NewWrapperBuilder & Cleaner() - just pass in volName and those can figure out the wrappee's name.